devices/ikea.js

```diff
@@ -1025,4 +1025,30 @@ module.exports = [
         description: 'PILSKOTT LED floor lamp',
         extend: tradfriExtend.light_onoff_brightness(),
     },
+    {
+        zigbeeModel: ['VINDSTYRKA'],
+        model: 'E2112',
+        vendor: 'IKEA',
+        description: 'Vindstyrka Air Quality and Humidity sensor',
+        fromZigbee: [fz.temperature, fz.humidity, fz.heiman_pm25],
+        toZigbee: [],
+        exposes: [e.temperature(), e.humidity(), e.pm25()],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const ep = device.getEndpoint(1);
+            await reporting.bind(ep, coordinatorEndpoint,
+                ['msTemperatureMeasurement', 'msRelativeHumidity', 'pm25Measurement']);
+            await ep.configureReporting('msTemperatureMeasurement', [{
+                attribute: 'measuredValue',
+                minimumReportInterval: 60, maximumReportInterval: 120,
```

Comment:
The max reporting interval seems rather short for all of these, setting the `change` to `50` (0.5C) or maybe `25` (0.25C) and increasing this to something like 900 or even an HOUR. (If this is not a battery powered device, it's probably fine to upgrade very often)

Reply:
The IKEA device does not respect the `change` settings on none of the provided clusters.

Since the device is not battery powered, I've decided that a minute reporting interval should be frequent enough for most use cases, but not too frequent to cause significant network traffic.